client/src/modules/Mentor/components/ReviewRandomTask/ReviewRandomTask.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button, message } from 'antd';
+import { useLoading } from 'components/useLoading';
+import { MentorsApi } from 'api';
+
+interface Props {
+  mentorId: number;
+  courseId: number;
+  onClick: () => void;
+}
+
+function ReviewRandomTask({ mentorId, courseId, onClick }: Props) {
+  const [loading, withLoading] = useLoading(false, e => {
+    const {
+      response: { status },
+    } = (e as any) || {};
+
+    if (status >= 400 && status < 500) {
```

Comment:
I understand your idea here but let's avoid working with status codes this way, IMHO they should be treated as a discrete values, not as a regular numbers

Reply:
I fixed a bit here. What is you suggestion regarding to "discrete values"? Could you please share some examples?